mxd/postman/management_api_tests.json

```diff
@@ -259,125 +350,181 @@
+					"raw": "{\n  \"@context\": {\n    \"odrl\": \"http://www.w3.org/ns/odrl/2/\"\n  },\n  \"@type\": \"NegotiationInitiateRequestDto\",\n  \"connectorAddress\": \"{{PROVIDER_PROTOCOL_URL}}\",\n  \"protocol\": \"dataspace-protocol-http\",\n  \"connectorId\": \"{{PROVIDER_ID}}\",\n  \"providerId\": \"{{PROVIDER_ID}}\",\n  \"offer\": {\n    \"offerId\": \"{{OFFER_ID}}\",\n    \"assetId\": \"{{ASSET_ID}}\",\n    \"policy\": {\n      \"@type\": \"odrl:Set\",\n      \"odrl:permission\": {\n        \"odrl:action\": {\n          \"odrl:type\": \"USE\"\n        },\n        \"odrl:constraint\": {\n          \"odrl:or\": {\n            \"odrl:leftOperand\": \"BusinessPartnerNumber\",\n            \"odrl:operator\": {\n              \"@id\": \"odrl:eq\"\n            },\n            \"odrl:rightOperand\": \"{{CONSUMER_ID}}\"\n          }\n        }\n      },\n      \"odrl:prohibition\": [],\n      \"odrl:obligation\": [],\n      \"odrl:target\": \"{{ASSET_ID}}\"\n    }\n  }\n}",
 					"options": {
 						"raw": {
 							"language": "json"
 						}
 					}
 				},
 				"url": {
-					"raw": "{{BOB_MANAGEMENT_URL}}/v2/policydefinitions/request",
+					"raw": "{{CONSUMER_MANAGEMENT_URL}}/v2/contractnegotiations",
 					"host": [
-						"{{BOB_MANAGEMENT_URL}}"
+						"{{CONSUMER_MANAGEMENT_URL}}"
 					],
 					"path": [
 						"v2",
-						"policydefinitions",
-						"request"
+						"contractnegotiations"
 					]
 				}
 			},
 			"response": []
 		},
 		{
-			"name": "Get all Assets",
+			"name": "Get Negotations By ID",
 			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"const API_KEY_HEADER = 'X-Api-Key';\r",
+							"const URL=`${pm.collectionVariables.get(\"CONSUMER_MANAGEMENT_URL\")}/v2/contractnegotiations/${pm.collectionVariables.get(\"NEGOTIATION_ID\")}/state`;\r",
+							"function delay(ms) {\r",
+							"  return new Promise(resolve => setTimeout(resolve, ms));\r",
+							"}\r",
+							"\r",
+							"async function makeRequests() {\r",
+							"  for (let i = 0; i < 10; i++) {\r",
+							"    const response = await new Promise(resolve => {\r",
+							"      pm.sendRequest({\r",
+							"        url: URL,\r",
+							"        method: 'GET',\r",
+							"        header: {\r",
+							"          ...pm.request.headers.toObject(),\r",
+							"          [API_KEY_HEADER]: pm.collectionVariables.get(\"API_KEY_PASSWORD\"),\r",
+							"        },\r",
+							"      }, (error, response) => {\r",
+							"        resolve(response);\r",
+							"      });\r",
+							"    });\r",
+							"\r",
+							"    var data = response.json();\r",
+							"    if (data[\"edc:state\"] === \"FINALIZED\") {\r",
```

Comment:
Field name is `state` instead of `edc:state`

Reply:
I have updated from edc:state to state in all the required places.